src/theme/index.tsx

```diff
@@ -20,6 +20,14 @@ export const MEDIA_WIDTHS = {
   upToLarge: 1280,
 }
 
+export enum Z_INDEX {
```

Comment:
might be worth looking at existing uses of `z-index` (e.g. modal would be closer to 9999)

https://github.com/Uniswap/uniswap-interface/search?q=z-index

Reply:
i like the idea of putting it there though

Reply:
yeah something i'd like to discuss around CSS style guide but imo using too many z-index values just gets confusing. i feel like it's a bit like using `!important` - keeping things simple and constrained reduces complexity everywhere and then you don't need huge values